app/middleware/debug_toolbar.py

```diff
@@ -0,0 +1,15 @@
+import os
+
+
+def check_toolbar_env(request):
+    """ callback to check whether debug toolbar should be shown or not
+
+    for details see
+    https://django-debug-toolbar.readthedocs.io/en/latest/configuration.html#debug-toolbar-config  # pylint: disable=line-too-long
+    """
+    print(os.environ.get('APP_ENV'))
+    if os.environ.get('APP_ENV', 'prod') in ['local', 'dev']:
+        print('blublu')
```

Comment:
We should use the logging instead of print statement here with a meaningful message :-)

Reply:
:+1: leftover from development, removed it